frontend/ui/lib/widgets/user_status_widget.dart

```diff
@@ -28,6 +30,22 @@ class _UserStatusState extends State<UserStatusWidget> {
   Icon _icon = Icon(Icons.favorite_border);
   Color _color = Colors.black;
 
+  _showDialog(BuildContext context, PlaceVisit placeVisit, PlaceVisitService placeVisitService) {
+    VoidCallback continueCallBack = () => {
+          placeVisitService.deletePlaceVisit(placeVisit.tripid, placeVisit.id),
+          Navigator.pushNamed(context, Router.createTripViewRoute(placeVisit.tripid)),
```

Comment:
We probably want pop-and-push, since we shouldn't be able to navigate back to the old view.